thrust/async/copy.h

```diff
@@ -141,9 +144,12 @@ struct copy_fn final
 
 THRUST_INLINE_CONSTANT copy_detail::copy_fn copy{};
 
+/*! \endcond
+ */
+
 } // namespace async
 
-} // end namespace thrust
+} // namespace thrust
```

Comment:
Was this necessary?

As a heads up, these changes will conflict with #1464.

Reply:
Yes, I noticed, I'll resolve it when I refresh the branch.

It's not necessary but I've tended to fix these as a drive by.